### ISSUE TITLE
Issue: No code is here :]

### ISSUE THREAD

**Issue by Raketemensch** [xuv/RasPiVJ#1]
Are you going to be posting code at some point? Your project is very interesting to me, especially since it has a web interface, which would allow you to control what's being displayed on screen without displaying the actual _controls_, since it has a web interface....

**Comment by xuv** [comment]
Ha. Tanks for noticing. Yes. There will be code. The project is running on the Raspi I have and I've used it in a concert already. But it needs proper documentation to be released. 
Where and when do you plan to use it? I could be working on it this week. Your message is certainly bringing back motivation. Curious also where you found out about it.

**Comment by Raketemensch** [comment]
Well, I’m a musician, and have been working on getting some video stuff going for a couple of acts I’m in. One is an original rock band, the other is all electronic, neither make any money :]

I’ve tried running Avenue before, which worked well, but I’ve got a pico projector that is pretty much the same size as a pi in a case, and I’m planning to basically velcro them together to make a portable VJ kit. I figure I’ll also put together a battery pack (I’ve done the research on this, but not built one yet) and have a completely portable kit that I can drag to gigs and set up wherever it’ll fit.

Googling around for VJing stuff eventually led me here:

http://w.xuv.be/projects/raspivj http://w.xuv.be/projects/raspivj

I’m a fairly capable geek, so if you need help or whatever, just let me know. I know my way around linux and have been working in software for way too long now.

**Comment by Raketemensch** [comment]
PocketVJ also looks very cool, but while their interface is available on github, their pi image doesn’t seem to be downloadable anywhere. And even if it was, I’m afraid I’d install it and find everything to be in German :]

**Comment by xuv** [comment]
Hey. Very cool to know  you want to do a portable VJ thing. That's the point of this app. Having musicians do their own visual show at the same time as playing. Although I must say I plan to change the interface. I've found out by using it that it needs to be rethought a bit, although usable so far. That's maybe also what prevented me from pushing the code.
Since I know now tat you have Liunx capabilities, I can be lighter on the documentation. I was planning to release a Pi image also. As an easy to install thing. But this will come later, when project is a little more stable I guess. 
As you might have read I got inspired by PocketVJ. But didn't not try to install their app. It was laking features for my use. But they surely are nice people, so don't hesitate to drop them a mail.
